src/components/Collapsible/tests/Collapsible.test.tsx

```diff
@@ -17,22 +17,6 @@ describe('<Collapsible />', () => {
 
     const hidden = collapsible.find(ariaHiddenSelector);
     expect(hidden.exists()).toBe(true);
-    expect(collapsible.contains('content')).toBe(false);
-  });
-
-  it('does not render its children when going from open to closed', () => {
-    const Child = () => null;
-
-    const collapsible = mountWithAppProvider(
-      <Collapsible id="test-collapsible" open>
-        <Child />
-      </Collapsible>,
-    );
-
-    expect(collapsible.find(Child)).toHaveLength(1);
-    collapsible.setProps({open: false});
-    collapsible.simulate('transitionEnd');
-    expect(collapsible.find(Child)).toHaveLength(0);
```

Comment:
I couldn't see any great benefit to removing and adding back the content. So I am removing the tests that would test for this.